.cookiecutter/includes/setuptools/install_requires

```diff
@@ -1,2 +1,2 @@
-pyjwt
+python-jose[cryptography]
```

Comment:
python jose supports JWE.

We already use it on LMS for it's support for JWKs and the plan is to settle in just this one library for JWT related needs.